plugins/BEdita/Core/src/Model/Table/ObjectTypesTable.php

```diff
@@ -211,11 +230,27 @@ public function get($primaryKey, $options = [])
      */
     public function beforeRules(Event $event, EntityInterface $entity)
     {
-        if ($entity->isNew() && empty($entity->parent_id)) {
-            $entity->set('parent_id', self::DEFAULT_PARENT_ID);
+        if ($entity->isNew()) {
+            if (empty($entity->get('parent_id'))) {
+                $entity->set('parent_id', self::DEFAULT_PARENT_ID);
+            }
+            if (empty($entity->get('plugin'))) {
```

Comment:
This might cause problems with models that are in application and _not_ in any plugin! `plugin` should be allowed to be `null`.

We can replace these checks with a single `if (empty($entity->get('table'))`, that sets table to `BEdita/Core.Objects` by default.

I know we're not planning to support models outside of plugin scope, but dealing with `table` property also provides a simpler approach, and I think it's safer than setting `plugin` and `model` to their default values in parallel.

Reply:
ok, done